src/Altinn.App.Core/EFormidling/Implementation/EformidlingStatusCheckEventHandler.cs

```diff
@@ -35,9 +29,9 @@ public class EformidlingStatusCheckEventHandler : IEventHandler
         private readonly IHttpClientFactory _httpClientFactory;
         private readonly IMaskinportenService _maskinportenService;
         private readonly MaskinportenSettings _maskinportenSettings;
-        private readonly IX509CertificateProvider _x509CertificateProvider;
         private readonly PlatformSettings _platformSettings;
         private readonly GeneralSettings _generalSettings;
+        private readonly IClientDefinition _clientDefinition;
```

Comment:
Lets discuss if this is the right dependency. It works, but I'm wondering if we should create a new IMaskinportenTokenProvider interface instead leaving it up to the provider how to get the token. That way we could get rid of IMaskinportenService, MaskinportenSettings and IClientDefinition and move those to the implementation of the new interface.

Reply:
@prange takk for PR. Vi ser behovet for å endre dette og den belyste problemet, men vi har diskutert det internt og ønsker en større endring for å gjøre dette mer i tråd med hvordan vi ser maskinporten oppkoblinger fremover. Jeg forstod det slik at dere har en workaround og at det ikke er kritisk for dere å få inn akkurat denne endringen? I så fall lukker vi bare denne nå og kommer med en oppdatering for bruk av maskinporten token i en app. Om dette betyr problemer og dere er avhengig av denne så gi beskjed.